website/docs/reference/resource-configs/postgres-configs.md

```diff
@@ -111,19 +111,21 @@ models:
 
 The Postgres adapter supports [materialized views](https://www.postgresql.org/docs/current/rules-materializedviews.html) and refreshes them for every subsequent `dbt run` you execute. For more information, see [Refresh Materialized Views](https://www.postgresql.org/docs/15/sql-refreshmaterializedview.html) in the Postgres docs.
 
-Materialized views support the optional configuration `on_configuration_change` with the following values: 
+Materialized views support the optional configuration `on_configuration_change` with the following values:
+
 - `apply` (default) &mdash; attempts to update the existing database object if possible, avoiding a complete rebuild. The following index action can be applied without the need to rebuild the materialized view:
-  - Added
-  - Dropped
-  - Updated
+    - Added
```

Comment:
If we're using `dropped`, should this be `created`? I feel like `drop` and `create` go together, or `add` and `delete`. And then I'm on the fence about `updated`, which is just a `drop` followed by a `create`; you can't alter an index as far as I can tell, so `alter` would be misleading.